api/python/indigo/__init__.py

```diff
@@ -2541,6 +2543,22 @@ def iterateBonds(self):
             ),
         )
 
+    def logP(self, csv_file=None) -> float:
+        """Molecule method returns calculated logP value
+
+        Returns:
+            float: calculated logP value of the molecule
+        """
+        return get_logp(self, csv_file=None)
+
+    def MR(self, csv_file=None) -> float:
```

Comment:
Thanks for adding this too.

Let's rename it to "molarRefractivity", "MR" sounds unclear.